assets/js/base/components/price-slider/stories/index.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { React, useState } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import PriceSlider from '../';
+
+export default {
+	title: 'WooCommerce Blocks/@base-components/PriceSlider',
+	component: PriceSlider,
+};
+
+export const Default = () => {
+	// PriceSlider expects client to update min & max price
+	// is this a bug? not fully self-contained
+	const [ min, setMin ] = useState( 1000 );
+	const [ max, setMax ] = useState( 5000 );
```

Comment:
Not a bug. The price slider is a controlled component.

Reply:
Thanks for explaining, tweaked my comment here: 0caa6ad3aff81640d4b5911620411053779927b5. Over time we can work to document these details about our components so it's clear how they should be used. Not sure what the conventions are here, perhaps there's a pattern we can follow, or maybe just document near `PropTypes`.

Reply:
I'd like to see more inline js docs for our components. It's something that we don't do a super great job at imo.  However, I think it's more important for publicly exposed components. It could be an effort that accompanies when we write stories.